api/views.py

```diff
@@ -291,7 +291,18 @@ def handle_post(self, request, *args, **kwargs):
         username = body['username']
         password = body['password']
 
-        user = authenticate(username=username, password=password)
+        #allowing different lower/uppercase lettered usernames:
+        caseSensitiveUsername = username
+        try:
+            findUser = User._default_manager.get(username__iexact=username)
+        except User.DoesNotExist:
+            findUser = None
+        if findUser is not None:
+            caseSensitiveUsername = findUser
+
+        #user = authenticate(username=username, password=password)
```

Comment:
@szabozoltan69 please remove this unless you have a reason to keep it here. If you're worried about reverting this change, that should be handled by proper git history.

Reply:
It seemed to be reasonable to keep here the "clean" (case sensitive) solution also. But it can be deleted also if it results nicer screen.

Reply:
I don't agree. This code is outdated. It has been replaced by your solution. There is no reason to keep it when it already exists in git history.